init_milvus.py

```diff
@@ -0,0 +1,110 @@
+import os
+import time
+import glob
+from dotenv import load_dotenv
+from langchain_text_splitters import RecursiveCharacterTextSplitter
+from pymilvus import FieldSchema, CollectionSchema, DataType, utility, Collection, connections, model
+
+load_dotenv()
+
+
+def read_md_files(directory):
+    md_files = glob.glob(f"{directory}/**/*.md", recursive=True)
+    docs = []
+    for file in md_files:
+        with open(file, "r", encoding="utf-8") as f:
+            docs.append(f.read())
+    return docs
+
+def split_md_by_sections(content):
```

Comment:
Yes, the Zilliz UI does not provide a complete view of the embeddings. I have added a functionality where the file name and the section start of each split is logged to stdout. The sections are written to the tmp/sections_split folder along with the file name and the section start as their headers so that we can easily see how our data is being split. For now, the data is being split into 161 sections.